idaes/models/properties/modular_properties/state_definitions/FTPx.py

```diff
@@ -423,7 +418,7 @@ def state_initialization(b):
             ) = identify_VL_component_list(b, pp)
             pp_VLE = pp
 
-    if init_VLE:
+    if num_VLE == 1:  # Only support initialization when a single VLE is present
```

Comment:
Should there be an `else` clause if num_VLE somehow exceeds 1, where either an exception is raised or a warning is logged?